src/plugins/discover/server/ui_settings.ts

```diff
@@ -242,15 +243,27 @@ export const getUiSettings: () => Record<string, UiSettingsParams> = () => ({
     category: ['discover'],
     schema: schema.boolean(),
   },
+  [ROW_HEIGHT_OPTION]: {
+    name: i18n.translate('discover.advancedSettings.params.rowHeightTitle', {
+      defaultMessage: 'Maximum lines of document explorer cell',
+    }),
+    value: 3,
+    category: ['discover'],
+    description: i18n.translate('discover.advancedSettings.params.rowHeightText', {
+      defaultMessage:
+        'The maximum lines that a cell in a table should occupy. Auto height might be enabled by setting value to zero.',
```

Comment:
This could be misleading, so users could think that if e.g. the content has 1 line, just 1 line is displayed. But if the users configures e.g. 4 lines , there are always 4 lines displayed, no matter how much content is available in the cell. adding @gchaps for finding good wording